single_tech_samples/synapseanalytics/sample3_integration_testing/synapse/notebook/ETL_sample.ipynb

```diff
@@ -0,0 +1,132 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
```

Comment:
is this a typo? i don't know if it should say "interact" or if this is dynamically generated

Reply:
looks like its consistently "nteract" across this file. likely not a typo, if not - please feel free to resolve this conversation :)

Reply:
Not a typo - the python notebooks were downloaded from Synapse and moved here